config/initializers/event_bus.rb

```diff
@@ -22,7 +22,8 @@
 
   # send bulk emails after events
   Event::BULK_MAIL_KINDS.each do |kind|
-    config.listen("#{kind}_event") { |event| SendBulkEmailJob.perform_later(event.id) }
+    delay = ENV.fetch('BULK_MAIL_DELAY', 0).to_i.seconds
+    config.listen("#{kind}_event") { |event| SendBulkEmailJob.set(wait: delay).perform_later(event.id) }
```

Comment:
Actually, that might not be quite right. But it does throw DeserializationError when trying to reconstruct the event object, so finding it by id was easier.